.github/workflows/readme.yml

```
@@ -6,58 +6,267 @@ on:
     branches:
       - main
       - master
-      paths:
-        - 'README.md'  # Only trigger on README.md changes
-      paths-ignore:
-        - 'docs/**'    # Ignore changes in docs directory
+    paths:
+      - 'README.md'  # Only trigger on README.md changes
+    paths-ignore:
```

actionlint (1.7.7)

both "paths" and "paths-ignore" filters cannot be used for the same event "push". note: use '!' to negate patterns

(events)